index.html

```diff
@@ -1,160 +1,163 @@
-<!doctype html>
-<html lang="en">
-  <head>
-    <meta charset="utf-8">
-    <meta http-equiv="X-UA-Compatible" content="chrome=1">
-    <title>Systers Open Source Projects by Systers</title>
-
-    <link rel="stylesheet" href="/stylesheets/styles.css">
-    <link rel="stylesheet" href="/stylesheets/fontawesome-all.css">
-    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no">
-    <link rel="icon" href="/images/favicon.png">
-  </head>
-
-  <body>
-    <header>
-    <a href="#main-menu" 
-        class="menu-toggle"
-        role="button"
-        id="main-menu-toggle"
-        aria-expanded="false"
-        aria-controls="main-menu"
-        aria-label="Open main menu">
-        <span class="sr-only">Open main menu</span>
-        <span class="fa fa-bars hamburger" aria-hidden="true"></span>
-    </a>
-
-    <div>
-            <a href="/"><img class="logo" src="/images/systers-logo.png" 
-                          alt="Systers - Anitab.org logo"></a>
-    </div>
-
-  <nav id="main-menu" 
-        class="main-menu"
-        role="navigation"
-        aria-expanded="false"
-        aria-label="Main menu">
-    <a href="#main-menu-toggle" 
-        class="menu-close"
-        role="button"
-        id="main-menu-close"
-        aria-expanded="false"
-        aria-controls="main-menu"
-        aria-label="Close main menu">
-        <span class="sr-only">Close main menu</span>
-        <span class="fa fa-close" aria-hidden="true"></span>
-    </a>
-    <ul>
-        <li><a href="#join-systers">Join Systers</a></li>
-        <li><a href="#projects">Projects</a></li>
-        <li><a href="#programs">Programs</a></li>
-        <li class="toggle"><a href="#main-menu">Contribute<span class="fas fa-chevron-down"></span></a>
-            <ul>
-              <li><a href="/newcomers">Newcomers</a></li>
-              <li><a href="/open-source-workflow">Open Source Workflow</a></li>
-              <li><a href="/member-levels">Member Levels</a></li>
-            </ul>
-        </li>
-            <li class="toggle"><a href="#main-menu">About Us <span class="fas fa-chevron-down"></span></a>
-            <ul>
-              <li><a href="/contactus.html">Contact Us</a></li>
```

Comment:
@Vertika-Shukla Looks like this was the only change that was intended. But your text editior auto-modified the whole file. Not a good practise.